specification/attributes/column_naming_convention.md

```diff
@@ -0,0 +1,34 @@
+# Column naming convention
+
+Column IDs provided in billing datasets following a consistent naming convention reduces friction for FinOps
+practitioners that consume the datasets for analysis, reporting, and other use cases.
+
+All columns defined in the FOCUS specification MUST follow the naming requirements listed below. Provider generated
+columns SHOULD adopt these same naming requirements over time.
+
+## Attribute ID
+
+ColumnNamingConvention
+
+## Attribute name
+
+Column Naming Convention
+
+## Description
+
+Naming Convention for columns appearing in billing data.
+
+## Requirements
```

Comment:
Each of these bullets should end in a period.

Reply:
My recommendation is the AP Style as per: https://www.instructionalsolutions.com/blog/bulleted-list-punctuation

- Use a period or other full stop after every bullet that is a sentence.
- Use a period after bullet list that completes the opening stem sentence that introduces it.
- Don’t use a period after bullet lists that are not complete sentences or do not complete the opening stem sentence. 
- Don’t use semicolons to end punctuation. 
- Use either all full sentences in your bullet lists or all fragments. Avoid a mix.

Reply:
So in this case, yes to full stops.